public/css/shell.css

```diff
@@ -0,0 +1 @@
+@charset "UTF-8";body{margin:0}body,div,html,ul{border:0;font-family:inherit;font-size:100%;font-style:inherit;font-weight:inherit;margin:0;outline:#000;padding:0;vertical-align:baseline}html{font-size:62.5%;overflow-y:scroll}body{background-color:#fff;background-position:initial initial;background-repeat:initial initial}ul{list-style:none}.wpcom-site__logo{color:#c8d7e1;font-size:12vw;position:fixed;top:50%;left:50%;-webkit-transform:translate(-50%,-50%)}@media (min-width:961px){.wpcom-site__logo{font-size:120px}}.wpcom-site__global-noscript{position:fixed;bottom:0;left:0;right:0;padding:6px;color:#fff;background-color:rgba(46,68,83,.8);text-align:center;z-index:300000;background-position:initial initial;background-repeat:initial initial}.wpcom-site,body,html{height:100%}body{background-color:#f3f6f8;background-position:initial initial;background-repeat:initial initial}body{font-family:-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen-Sans,Ubuntu,Cantarell,'Helvetica Neue',sans-serif}body{color:#2e4453;font-size:15px;line-height:1.5}noscript{text-align:center;margin-top:3em;display:block}ul{margin:0 0 1.5em 3em}ul{list-style:disc}.noticon::after,.noticon::before{-webkit-font-smoothing:antialiased;font-family:Noticons;line-height:1}.layout__content{position:relative;margin:0;padding:79px 32px 32px 304px;box-sizing:border-box;overflow:hidden}.layout__content::after{content:'.';display:block;height:0;width:0;clear:both;visibility:hidden}@media (max-width:960px){.layout__content{padding:71px 24px 24px 252px}}@media (max-width:660px){.layout__content{margin-left:0;padding:47px 0 0}}.layout .sidebar{-webkit-transform:translateX(0)}.masterbar{background-color:#0087be;border-bottom-width:1px;border-bottom-style:solid;border-bottom-color:#0079aa;color:#fff;font-size:16px;display:-webkit-flex;height:46px;margin:0;position:fixed;left:0;top:0;width:100%;z-index:180;-webkit-font-smoothing:subpixel-antialiased;background-position:initial initial;background-repeat:initial initial}@media (min-width:661px){.masterbar{-webkit-backface-visibility:hidden}}.sidebar{display:-webkit-flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-webkit-flex-direction:column;overflow:auto;padding:0;background-color:#e9eff3;position:fixed;top:47px;bottom:0;left:0;background-position:initial initial;background-repeat:initial initial}@media (max-width:960px){.sidebar{border-right-width:1px;border-right-style:solid;border-right-color:#d9e3ea;width:228px}}@media (min-width:961px){.sidebar{border-right-width:1px;border-right-style:solid;border-right-color:#d9e3ea;width:272px}}@media (max-width:660px){.sidebar{left:-100%;width:100%;max-height:calc(100% - 47px);-webkit-transform:translateX(0)}}.sidebar{margin:0}.is-group-reader .sidebar{overflow-x:hidden}@media (max-width:660px){.is-group-reader .sidebar{z-index:23}}
```

Comment:
Neverming, I've just seen that this must be generated while the server is running.